index.php

```diff
@@ -76,5 +76,13 @@
         
         <div ng-view class="view-animate"></div>
     </body>
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
+      ga('create', 'UA-59504141-1', 'auto');
+
+    </script>

```

Comment:
if we want to keep this extra clean we could put this in a separate js file and include it in one line so index.php stays pretty sparse.